lib/response.js

```diff
@@ -520,5 +522,12 @@ module.exports = {
       message: this.message,
       header: this.header
     };
+  },
+
+  /**
+   * Flush any set headers, and begin the body
+   */
+  flushHeaders: function(){
+    this.res.write('');
```

Comment:
@dougwilson that was my first thought too, but if I add `this.res.statusCode = 200;` in directly before the `writeHead` it gives me the same error.